tests/test_config_parser.py

```diff
@@ -185,14 +190,14 @@ def _append_list(self, ref_entry, new_args, job_name=DEFAULT_JOB_NAME, separator
                 self.reference_config[job_name][ref_entry].append(arg)
 
     @staticmethod
-    def write_ini(dname: str, fname: str, config: dict, prio=1000) -> None:
+    def write_ini(dname: str, fname: str, config: Dict[str, Any], prio: int = 1000) -> None:
         """Write mkosi.default(.d/*) files"""
         if not os.path.exists(dname):
             os.makedirs(dname)
         if prio < 1000:
             fname = f"{prio:03d}_{fname}"
         config_parser = configparser.RawConfigParser()
-        config_parser.optionxform = lambda optionstr: str(optionstr)
+        config_parser.optionxform = lambda optionstr: str(optionstr) # type: ignore
```

Comment:
The discussed solutions seemed to be `# type ignore` and `setattr`

Reply:
Ignoring is fine here, but add a comment linking to that discussion so we know why we added the ignore in the future.